.github/workflows/create-tx-for-multisig.yml

```
@@ -0,0 +1,67 @@
+name: Build TX for Multisig Operation
+
+on:
+  workflow_dispatch:
+    inputs:
+      transaction_type:
+        description: "select transaction type to create"
+        type: choice
+        options:
+          - upgrade-subsidies
+          - migrate-subsidies
+      rpc:
+        description: "RPC url"
+        required: true
+        default: "https://fullnode.mainnet.sui.io:443"
+        type: string
+      gas_object_id:
+        description: "object id to get gas from for multisig transaction"
+        required: false
+        type: string
+      commit:
+        description: "commit hash of the walrus repo to create the transaction"
+        required: true
+        type: string
+
+jobs:
+  create-tx:
+    name: Create TX for multisig
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Selected transaction type
+        run: |
+          echo ${{ inputs.transaction_type }}
+
+      - uses: actions/checkout@v4
```

Nit: Can we pin the version?
```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # pin@v4
```